js/README.md

```diff
@@ -32,8 +33,8 @@ Please follow the steps described below to setup development environment.
 - Visual Studio Code: https://code.visualstudio.com/
 
   - **required** extension: [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
-  - **required** extension: [Clang-Format](https://marketplace.visualstudio.com/items?itemName=xaver.clang-format)
-  - **required** extension: [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
+  - **required** extension: [Prettier](https://marketplace.visualstudio.com/items?itemName=SimonSiefke.prettier-vscode)
```

Comment:
May be marked it as optional since 'npm run lint' and 'npm run format' shall work, and we could have better control of the version installed by `npm ci`.

What if the extension has different format result since version of eslint or prettier of extension are not fixed?

Reply:
The VSCode config already have the settings for the extension to use the version that installed from `npm ci`.